pkg/operator/ceph/object/objectstore.go

```diff
@@ -38,6 +43,8 @@ const (
 	// AppName is the name Rook uses for the object store's application
 	AppName               = "rook-ceph-rgw"
 	bucketProvisionerName = "ceph.rook.io/bucket"
+	accessKeyName         = "access-key"
+	secretKeyName         = "secret-key"
```

Comment:
accessKeyName and secretKeyName were already defined in another place. Can we reference one of them instead of duplicating?